openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -868,3 +869,17 @@ export const getUniqueArray = (count: number) =>
   [...Array(count)].map((_, index) => ({
     key: `key${index}`,
   }));
+
+/**
+ * @param searchValue search input
+ * @param option select options list
+ * @returns boolean
+ */
+export const handleSearchFilterOption = (
+  searchValue: string,
+  option?: {
+    label: string;
+    value: string;
+  }
+) => lowerCase(option?.label).includes(lowerCase(searchValue));
```

Comment:
The `lowerCase` will convert the string into space separated lowercase string. so instead use `toLowerCase()` native sting method.

example:

```js
lowerCase('fooBar');
// => 'foo bar'
```

Reply:
[toLower](https://lodash.com/docs/4.17.15#toLower) is also an option here